src/index.ts

```diff
@@ -1,6 +1,7 @@
 import app from './bootstrap';
 import Kernel from './bootstrap/Kernel';
 
+// eslint-disable-next-line no-console
```

Comment:
This line can be removed, it was part of some of the debugging stuff.

Reply:
Removed